package.json

```diff
@@ -89,7 +89,7 @@
     "conventional-changelog-cli": "^2.0.11",
     "conventional-changelog-videojs": "^3.0.0",
     "es5-shim": "^4.5.10",
```

Comment:
I have been wondering this, can we remove es5-shim at this point?

Reply:
Maybe, it doesn't hurt to keep it in. It's only used for tests and helps make sure everything is up-to-spec. Since we're loading in es6-shim and es7-shim, it's probably best to keep es5-shim as well.